index.html

```diff
@@ -6470,26 +6470,12 @@ <h3>Executing Script</h3>
 <dl class=switch>
  <dt><a>undefined</a>
  <dt><a><code>null</code></a>
- <dd><p><a>Success</a> with data <a><code>null</code></a>.
+ <dd><p><p>Return <a>success</a> with data set to <a><code>null</code></a>.
```

Comment:
We just say "success with data [foo]", not "data set to [foo]" (here and throughout).

Reply:
Sure. Will push an update.